### PR TITLE
fix description label bugs

### PR DESCRIPTION
**Pull request by dkent600**
Resolves: https://github.com/daostack/alchemy/issues/1942

Note this is already fixed in 2.0

### CODE REVIEW

File: src/components/Proposal/Create/SchemeForms/CreateKnownGenericSchemeProposal.tsx [hunk 1]
```diff
@@ -286,6 +286,7 @@ class CreateKnownSchemeProposal extends React.Component<IProps, IState> {
 
     const actions = this.state.actions;
     const currentAction = this.state.currentAction;
+    const fnDescription = () => (<span>Short description of the proposal.<ul><li>What are you proposing to do?</li><li>Why is it important?</li><li>How much will it cost the DAO?</li><li>When do you plan to deliver the work?</li></ul></span>);
```
Comment (roienatan) [inline src/components/Proposal/Create/SchemeForms/CreateKnownGenericSchemeProposal.tsx:289]: Is there a place we can hold constant text like this?
The code can look cleaner if it'll will be fetched from a constants file for example.
Also, I think we use this text in more places so moving this to a separate file can reduce code duplication.

Comment (dkent600) [reply src/components/Proposal/Create/SchemeForms/CreateKnownGenericSchemeProposal.tsx:289]: > Is there a place we can hold constant text like this?

I would like to do that too, this is often-duplicated code.  I'd say in proposalUtils.  But not in this PR.

Comment (orenyodfat) [reply src/components/Proposal/Create/SchemeForms/CreateKnownGenericSchemeProposal.tsx:289]: agree , lets move it to a some lable.json somehow.

Comment (dkent600) [reply src/components/Proposal/Create/SchemeForms/CreateKnownGenericSchemeProposal.tsx:289]: in a separate PR.  not needed in 1.0

Comment (dkent600) [reply src/components/Proposal/Create/SchemeForms/CreateKnownGenericSchemeProposal.tsx:289]: in any case, I think putting this in a json file may be overkill, over-engineering unless we can show a need for it.

Comment (dkent600) [reply src/components/Proposal/Create/SchemeForms/CreateKnownGenericSchemeProposal.tsx:289]: note that fnDescription  needs to be a component...

Comment (orenyodfat) [reply src/components/Proposal/Create/SchemeForms/CreateKnownGenericSchemeProposal.tsx:289]: moving all texts to specific file  is much cleaner code and well engineered.
will handle that on 2.0.

**Pull request merged**